hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/utils/DelegationSASGenerator.java

```diff
@@ -82,21 +85,22 @@ public String getDelegationSAS(String accountName, String containerName, String
       case SASTokenProvider.LIST_OPERATION:
         sp = "l";
         break;
-    case SASTokenProvider.LIST_BLOB_OPERATION:
+      case SASTokenProvider.LIST_BLOB_OPERATION:
```

Comment:
Spacing issue

Reply:
There was a spacing issue earlier, I fixed it in this change.